docs/navigation-parameters-reference.md

```diff
@@ -865,6 +865,45 @@ The app switcher is a drop-down list available in the top navigation bar. It all
 - **type**: boolean
 - **description**: includes the link to the root of the Luigi application in the drop-down using the **title** specified in the **settings/header** section of the configuration as a label.
 
+### customAppSwitcherItemRenderer
+- **type**: Function
+- **description**: This function allows you to customize the single list element rendered in the default app switcher popover.
+- **attributes**:
+  - **item** single application element
+  - **slot** `ul` element as slot. You can append your custom `li` entries to this `ul` element.
```

Comment:
One more small thing: since the "rule" now is for parameters to be in alphabetical order, and this one begins with `c`, it should be around line 855 instead of 868. But I'm not able to make this edit in a GitHub suggestion  😄

```suggestion
### customAppSwitcherItemRenderer
- **type**: function
- **description**: This function allows you to customize the single list element rendered in the default app switcher popover.
- **attributes**:
  - **item** single application element.
  - **slot** `ul` element as slot. You can append your custom `li` entries to this `ul` element.
```

Reply:
Thx Alex. Good catch!